lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -1640,24 +1578,19 @@ class MM600SeriesDriver extends BCNLDriver {
     return new PumpStatusCommand(this.pumpSession).send(this.hidDevice);
   }
 
-  getBolusWizardSettings() {
-    // eslint-disable-next-line no-unused-vars
-    return new Promise((resolve, reject) => {
-      const settings = {};
-      new BolusWizardBGTargetsCommand(this.pumpSession).send(this.hidDevice)
-        .then((response) => {
-          settings.bgTarget = response.targets;
-        })
-        .then(() => new BolusWizardCarbRatiosCommand(this.pumpSession).send(this.hidDevice))
-        .then((response) => {
-          settings.carbRatio = response.ratios;
-        })
-        .then(() => new BolusWizardSensitivityFactorsCommand(this.pumpSession).send(this.hidDevice))
-        .then((response) => {
-          settings.insulinSensitivity = response.factors;
-        })
-        .then(() => resolve(settings));
-    });
+  async getBolusWizardSettings() {
+    const settings = {};
+    let response = await new BolusWizardBGTargetsCommand(this.pumpSession).send(this.hidDevice);
+    settings.bgTarget = response.targets;
+
+    response = await new BolusWizardCarbRatiosCommand(this.pumpSession).send(this.hidDevice);
+    settings.carbRatio = response.ratios;
+
+    response =
+      await new BolusWizardSensitivityFactorsCommand(this.pumpSession).send(this.hidDevice);
```

Comment:
It breaks the eslint rule for max line size. But it wasn't as readable as it could be, so I'll change it to:
```
response = await new BolusWizardSensitivityFactorsCommand(this.pumpSession)
      .send(this.hidDevice);
```